src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1246,9 +1246,15 @@
   - name: turn on "alwaysOn"
     text: >
         az webapp config set -g MyResourceGroup -n MyUniqueApp --always-on true
-  - name: turn on "alwaysOn" through a json with content '{"alwaysOn", true}'
+  - name: for non-PowerShell users, turn on "alwaysOn" through a json with content '{"alwaysOn", true}'
     text: >
         az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations '{"alwaysOn": true}'
+  - name: for PowerShell users, turn on "alwaysOn" through a json with content '{"alwaysOn", true}' (note the backslashes escaping the double quotes)
+    text: >
+        az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations '{\"alwaysOn\": true}'
+  - name: for PowerShell users, set configuration through a JSON file called params.json
+    text: >
+        az webapp config set -g MyResourceGroup -n MyUniqueApp --generic-configurations "@.\\params.json"
```

Comment:
I don't think it matters, but why are there 2 "\" in the path to the JSON? I just tried it and it works, but 1 "\" is enough.